ios/iosApp/UI/BaseViewModelView.swift

```diff
@@ -0,0 +1,15 @@
+import TrikotFrameworkName
+import UIKit
+
+class BaseViewModelView<VM: ViewModel>: UIView {
+    var viewViewModel: VM?
+
+    override required init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
```

Comment:
Doing it like this removes the need to always override `init(frame:)` and being stuck with override `init(coder:)` everywhere:
```
    required init(viewModel: VM) {
        self.viewViewModel = viewModel
        super.init(frame: .zero)
        postInit()
        setupBindings(viewModel: viewModel)
    }

    func postInit() {
    }

    func setupBindings(viewModel: VM) {
    }

    @available(*, unavailable)
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
```

Reply:
And removes the need for setting the viewModel after construction

Reply:
I really dispprove / like the pattern

Reply:
Good point too! I just haven't encountered the case in my projet, disregard my comment. 
We really need finer control on when stuff is binded